api-connectors/amadeus/interest.json

```diff
@@ -0,0 +1,57 @@
+{
+    "version": 1,
+    "request": {
+        "url": "https://test.api.amadeus.com/v1/reference-data/locations/pois",
+        "method": "GET",
+        "authorization": {
+            "type": "OAuth2",
+            "grantType": "ClientCredentials",
+            "tokenServerUrl": "https://test.api.amadeus.com/v1/security/oauth2/token"
+        },
+        "params": {
+            "latitude": true,
+            "longitude": true,
+            "radius": false,
+            "limit": {
+                "required": false,
+                "removeIfEmpty": true,
+                "fromKey": "limit",
+                "toKey": "page[limit]"
+            },
+            "page[limit]": false
+        },
+        "pagination": {
+            "type": "offset",
+            "offsetKey": "page[offset]",
+            "limitKey": "page[limit]",
+            "maxCount": 25
+        }
```

Comment:
I tried to use `limit` parameter, but it makes no sense.
```
df = await dc.query('interest', latitude=41.397158, longitude=2.160873, limit=3)
```
This was what I tried, I still got 10 rows of the data.